### PR TITLE
puffypcp Cython extension

### PR DESCRIPTION
**Pull request by iMurfyD**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -60,4 +59,4 @@ include config/templates/pmie/control
 include config/templates/pmie/pcp-restart.sh
 include config/templates/pmie/pmie-supremm.config
 include assets/modw_supremm.sql
-include assets/mongo_setup.js
+include assets/mongo_setup.j
```
Comment (jpwhite4) [inline MANIFEST.in]: Whoops. Need to not change this line.

File: supremm/statistics.py [hunk 2]
```diff
@@ -14,7 +13,7 @@ def calculate_stats(v):
 
         if v_min == v_max:
             return {'avg': float(v[0]), 'cnt': len(v)}
-
+
```
Comment (jpwhite4) [inline supremm/statistics.py]: Need to not have these whitespace changes in the pull request.

File: supremm/puffypcp/pcp.pxd [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        pmID pmid
+        int numval
+        int valfmt
+        pmValue vlist[1]
+    ctypedef struct pmResult:
+        timeval timestamp
+        int numpmid
+        pmValueSet *vset[1]
+    ctypedef union pmAtomValue:
+        # TODO use <inttypes.h> types instead of simple long etc.
```
Comment (jpwhite4) [inline supremm/puffypcp/pcp.pxd]: Will this code run correctly on a 32bit machine?

Comment (iMurfyD) [inline supremm/puffypcp/pcp.pxd]: Untested, but I suspect it should. The pxd file is just a placeholder for Cython to know what is valid C code. Those definitions are pulled directly from the pcp headers.

**Comment by jpwhite4** [comment]
These code changes were merged in #148

**Pull request closed without merging**